pkgs/applications/office/qownnotes/default.nix

```diff
@@ -60,7 +61,7 @@ stdenv.mkDerivation {
     changelog = "https://www.qownnotes.org/changelog.html";
     downloadPage = "https://github.com/pbek/QOwnNotes/releases/tag/v${version}";
     license = licenses.gpl2Only;
-    maintainers = with maintainers; [ totoroot ];
+    maintainers = with maintainers; [ pbek totoroot ];
```

Comment:
It seems that pbek is not in the [maintainer-list](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix).  @pbek  Would you like to add yourself to the maintainer list?

Reply:
Certainly, thank you for mentioning!
https://github.com/NixOS/nixpkgs/pull/227399